<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="4" hostname="Mahendra"
	name="Web_Browser.TestNG_Driver.Test_Driver" tests="5" failures="0"
	timestamp="06 Oct 2017 19:06:56 GMT" time="10.390" errors="1">
	<testcase name="launchapplication" time="10.390"
		classname="Web_Browser.TestNG_Driver.Test_Driver">
		<error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at Web_Browser.Login_Page.Login_Components.<init>(Login_Components.java:17)
	at Web_Browser.TestNG_Driver.Test_Driver.launchapplication(Test_Driver.java:20)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:523)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:224)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:146)
	at org.testng.TestRunner.beforeRun(TestRunner.java:658)
	at org.testng.TestRunner.run(TestRunner.java:626)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:425)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:420)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:385)
	at org.testng.SuiteRunner.run(SuiteRunner.java:334)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1318)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1243)
	at org.testng.TestNG.runSuites(TestNG.java:1161)
	at org.testng.TestNG.run(TestNG.java:1129)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
		</error>
	</testcase> <!-- launchapplication -->
	<testcase name="test2" time="0.000"
		classname="Web_Browser.TestNG_Driver.Test_Driver">
		<skipped />
	</testcase> <!-- test2 -->
	<testcase name="test1" time="0.000"
		classname="Web_Browser.TestNG_Driver.Test_Driver">
		<skipped />
	</testcase> <!-- test1 -->
	<testcase name="test3" time="0.000"
		classname="Web_Browser.TestNG_Driver.Test_Driver">
		<skipped />
	</testcase> <!-- test3 -->
	<testcase name="test4" time="0.000"
		classname="Web_Browser.TestNG_Driver.Test_Driver">
		<skipped />
	</testcase> <!-- test4 -->
</testsuite> <!-- Web_Browser.TestNG_Driver.Test_Driver -->
